src/components/FormFieldComponent.vue

```diff
@@ -152,26 +153,44 @@
     methods: {
       onDataChange () {
         this.$emit('dataChange')
-      },
-      // Can return more than only a boolean because of internationalized messages
-      validate () {
+      }
+    },
+    computed: {
+      /*
+       * Compute visibility to:
```

Comment:
copy paste error , this is not the 'visibility' comment, i suggest we remove these comments ( they are more relevant for the PR), as this is default working of vue